src/rsuite/SelectUI.tsx

```diff
@@ -273,26 +275,26 @@ export const SelectPopupUI = observer(function SelectPopupUI_<T>(p: { s: AutoCom
                     <li
                         key={index}
                         style={{ minWidth: '10rem' }}
-                        className={`p-0.5 hover:bg-base-300 cursor-pointer ${index === s.selectedIndex ? 'bg-base-300' : ''}`}
-                        tw={['flex items-center gap-1']}
+                        className={`active:bg-base-300 cursor-pointer ${index === s.selectedIndex ? 'bg-base-300' : ''}`}
+                        tw={[
+                            'flex items-center gap-1 rounded',
+                            isSelected && 'bg-primary text-primary-content hover:text-neutral-content',
+                        ]}
+                        onMouseEnter={(ev) => {
+                            s.setNavigationIndex(index)
+                        }}
                         onMouseDown={(ev) => s.onMenuEntryClick(ev, index)}
                     >
-                        <div>
+                        <div tw={'rounded p-1 h-6'}>
```

Comment:
nitpick: no need for `{` `}` here (don't fix those nitpicks, we don't really care)